main.tf

```diff
@@ -0,0 +1,91 @@
+locals {
+  dimensions = {
+    ClusterName = module.alarm_label.environment
+    ServiceName = module.alarm_label.id
+  }
+}
+
+module "alarm_label" {
+  source  = "justtrackio/label/null"
+  version = "0.26.0"
+
+  label_order = var.label_orders.ecs
+
+  context = module.this.context
+}
+
+module "cpu_average" {
+  source  = "terraform-aws-modules/cloudwatch/aws//modules/metric-alarm"
+  version = "5.1.0"
+
+  alarm_name          = "${module.alarm_label.id}-cpu-average"
```

Comment:
Either this or the ServiceName in line 4 are incorrect. In the ServiceName we are not having the environment. In the Cloudwatch alarms however we usually have it. Can you check which one needs adjustments?